plugin/PDFView.qml

```diff
@@ -141,6 +143,11 @@ SilicaFlickable {
             returnY.from = pt.y - deltaY
             returnY.to = pt.y
             quickScrollAnimation.pageTo = pageId
+            if (focusItem !== undefined) {
+                quickScrollAnimation.focusItem = focusItem
```

Comment:
I prefer to delegate to the scroll() function, the task to activate the focus animation because otherwise, the caller needs to set the focusItem property for both scrollAnimation and quickScrollAnimation. IMHO, the caller should not know that implementation internally uses two different scroll animators.